README.md

```diff
@@ -1,3 +1,39 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Market Overview
+
```

Comment:
- [ ] **Good job Carlos 🥳🎉👏**. In your readme dont forget to add the screenshot of the app to make it professional